include/libobmcsession/obmcsession.h

```diff
@@ -0,0 +1,181 @@
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2021 YADRO.
+ */
+
+#ifndef __OBMCSESSION_H__
+#define __OBMCSESSION_H__
+
+#if (defined __cplusplus)
+extern "C"
+{
+#endif
+
+#include <systemd/sd-bus.h>
+#include <libobmcsession/obmcsession_proto.h>
+
+/** @brief Constructs session manager
+ *
+ * @param[in] bus           - Handle to system dbus
+ * @param[in] objPath       - The Dbus service slug uniquely identifying the
+ *                            source of a session to create the appropriate
+ *                            items from. Service name template:
+ *                            'xyz.openbmc_project.Session.${slug}'.
+ * @param[in] type          - Type of all session items that will be created
+ * by the current instance.
+ * @return int              - session manager initialization status
```

Comment:
Please specify that it's an `errno-like status`, not just `status`. Now it looks like it returns a zero/non-zero (which is also true), but nobody could ever guess that it actually returns an errno value.
Alternatively you could specify:
```
@retval == 0 - Success
@retval != 0 - An errno-like value indicating the type of failure
```